src/main/java/org/prebid/server/analytics/reporter/greenbids/model/CommonMessage.java

```diff
@@ -0,0 +1,39 @@
+package org.prebid.server.analytics.reporter.greenbids.model;
+
+import org.prebid.server.auction.model.AuctionContext;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CommonMessage {
+
+    public String version;
+    public String auctionId;
+    public String referrer;
+    public double sampling;
+    public String prebid;
+    public String greenbidsId;
+    public String pbuid;
+    public String billingId;
+    public List<AdUnit> adUnits;
+    public Long auctionElapsed;
+
+    public CommonMessage(
+            AuctionContext auctionContext,
+            GreenbidsConfig greenbidsConfig,
+            Long auctionElapsed,
+            String greenbidsId,
+            String billingId
+    ) {
+        this.version = "2.2.0";
+        this.auctionId = auctionContext.getBidRequest().getId();
+        this.referrer = auctionContext.getBidRequest().getSite().getPage();
+        this.sampling = greenbidsConfig.getGreenbidsSampling();
+        this.prebid = "$prebid.version$"; // TODO: to fix
```

Comment:
self-review: currently searching where PBS version field could be extracted from to the payload

Reply:
Yes I've added `+server` suffix to prebid version. Hardcoded current release version 3.0.0 for the moment (gonna discuss ask PBS team how to properly refactor it). 

`private static final String PREBID_SERVER_VERSION = "3.0.0+server";`